src/routes/transactions/pipes/delete-transaction.validation.pipe.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable, PipeTransform } from '@nestjs/common';
+import { ValidateFunction } from 'ajv';
+import { GenericValidator } from '@/validation/providers/generic.validator';
+import { JsonSchemaService } from '@/validation/providers/json-schema.service';
+import { DeleteTransactionDto } from '@/routes/transactions/entities/delete-transaction.dto.entity';
+import {
+  DELETE_TRANSACTION_DTO_SCHEMA_ID,
+  deleteTransactionDtoSchema,
+} from '@/routes/transactions/entities/schemas/delete-transaction.dto.schema';
+
+@Injectable()
+export class DeleteTransactionDtoValidationPipe
+  implements PipeTransform<unknown, DeleteTransactionDto>
+{
+  private readonly isValid: ValidateFunction<DeleteTransactionDto>;
+
+  constructor(
+    private readonly genericValidator: GenericValidator,
+    private readonly jsonSchemaService: JsonSchemaService,
+  ) {
+    this.isValid = this.jsonSchemaService.getSchema(
+      DELETE_TRANSACTION_DTO_SCHEMA_ID,
+      deleteTransactionDtoSchema,
+    );
+  }
```

Comment:
Nitpick: missing newline between functions 🙂

Reply:
Added in 731824e9750feec78d152c4caa1b4c0d592b326b.